docker-compose.yaml

```
@@ -0,0 +1,71 @@
+        condition: service_healthy
+    env_file:
+      - .env
+
+  nginx:
+    build:
+      context: .docker/nginx
+    restart: unless-stopped
+    ports:
+      - ${NGINX_HOST_PORT}:80
+      # - ${NGINX_HTTPS_HOST_PORT}:443
+    depends_on: 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)